appcues/build.gradle

```diff
@@ -71,6 +71,7 @@ dependencies {
     implementation "androidx.core:core-ktx:1.9.0"
     implementation 'androidx.activity:activity-compose:1.6.1'
     implementation "androidx.browser:browser:1.4.0"
+    implementation "androidx.startup:startup-runtime:1.1.1"
```

Comment:
this is again needed so we can have our provider started to register the activity lifecycle monitor at every cold start of the app.